APR-Core/QuestHandler.lua

```diff
@@ -3283,68 +3283,68 @@ APR_QH_EventFrame:SetScript("OnEvent", function(self, event, ...)
 			-- GOSSIP HARDCODED
 			if (steps and steps["Gossip"]) then
 				if (steps["Gossip"] == 27373 or steps["Gossip"] == 34398) then
-					SelectGossipOption(1)
+					C_GossipInfo.SelectOptionByIndex(1)
 					APR1[APR.Realm][APR.Name][APR.ActiveMap] = APR1[APR.Realm][APR.Name][APR.ActiveMap] + 1
 					APR.BookingList["UpdateQuest"] = 1
 					APR.BookingList["PrintQStep"] = 1
 				end
 				if (steps["Gossip"] == 3433398) then
-					SelectGossipOption(2)
+					C_GossipInfo.SelectOptionByIndex(2)
 					APR1[APR.Realm][APR.Name][APR.ActiveMap] = APR1[APR.Realm][APR.Name][APR.ActiveMap] + 1
 					APR.BookingList["UpdateQuest"] = 1
 					APR.BookingList["PrintQStep"] = 1
 				end
 				if (steps["Gossip"] == 28202) then
 					APRGOSSIPCOUNT = APRGOSSIPCOUNT + 1
 					if (APRGOSSIPCOUNT == 1) then
-						SelectGossipOption(1)
+						C_GossipInfo.SelectOptionByIndex(1)
 					elseif (APRGOSSIPCOUNT == 2) then
 						if (APR.Race == "Gnome") then
-							SelectGossipOption(1)
+							C_GossipInfo.SelectOptionByIndex(1)
 						elseif (APR.Race == "Human" or APR.Race == "Dwarf") then
-							SelectGossipOption(2)
+							C_GossipInfo.SelectOptionByIndex(2)
 						elseif (APR.Race == "NightElf") then
-							SelectGossipOption(3)
+							C_GossipInfo.SelectOptionByIndex(3)

```

Comment:
I don't know if it's this value or -1 of the value (we keep it and we'll do a fix if ever we get a bug)